src/styles/utils/variables.scss

```
@@ -0,0 +1,9 @@
+$day-gap: 1px;
+$day-width: 100px;
+$each-day: mon 0,
+  tue ($day-width + $day-gap),
+  wed ($day-width + $day-gap) * 2,
+  thu ($day-width + $day-gap) * 3,
+  fri ($day-width + $day-gap) * 4,
+  sat ($day-width + $day-gap) * 5,
+  sun ($day-width + $day-gap) * 6;
```

better to move this ($day-width + $day-gap) to loop directly 
```suggestion
$each-day: mon 0,
  tue 1,
  wed 2,
  thu 3,
  fri 4,
  sat 5,
  sun 6;
```